### DISCUSSION TITLE
Incredibly clueless security question...

### DISCUSSION THREAD

**Discussion by taurosetau** [Ralim/IronOS#1316] [Q&A] [answered]
Emphasis on clueless!

Ok, so, are there any other operating systems called Iron and if not, does IronOS have any features which could appear to a web application as if it were a browser?

I ask, because I was reviewing past logins on an online payment portal, and two logins were supposedly made via "Iron OS" (as opposed to "IronOS" with no space).  Doesn't seem any transactions were made through those logins, but I removed them anyhow and went to look for more information. Ralim/IronOS was the only thing I could find, but is spelt without the space. Other logins from my regular setup listed the expected browser & OS whereas these entries only had Iron OS.

Anyhow, in the rough time period surrounding the dates of those logins, I had been approaching some soldering projects, but to my knowledge I never mucked around with any firmware or software for any of that, and am unsure that IronOS is even relevant to my iron (a Hakko). I suppose it is possible that I downloaded a bundle in anticipation of getting a Pinecil or TS100 as I did weigh up both, but as you can guess from my query, I'm so unfamiliar with all of this stuff that I strongly doubt I'd have even gone that far, let alone done more with it. Furthermore, I've never knowingly connected my iron to anything aside from the mains power supply.

Like I said, probably a very dumb question. I'm not alarmed as no transactions were made via these logins, they were a long time ago, nothing else sketchy seems to have occurred, and I should be able to review access to my WiFi & implement extra restrictions if need be, but figured it was best to ask anyhow.  I'll email customer services for the online account to if they can shed any light on the matter and update if appropriate.

Thank you for reading.

**Comment by Ralim** [comment, marked as answer]
Hello,

I do not know of any way that this could happen from this code base. Nothing here should act as a web browser or be able to show up like that. None of these devices can use wi-fi either.

The name Iron OS had probably been used elsewhere or perhaps by a library that another piece of software is using.


I would agree that if you don't recognise it changing password and removing existing signed in sessions would be super wise.

**Reply by taurosetau** [reply]
Thank you for the reassurance. Much appreciated.
